sword-server/pom.xml

```diff
@@ -56,6 +56,10 @@
 					<artifactId>servlet-api</artifactId>
 					<groupId>javax.servlet</groupId>
 				</exclusion>
```

Comment:
what's the sword server doing in this PR, and why didn't it have jena previously?

Reply:
This is actually excluding jena from sword-server, since it was importing an old version of jena.  It was mainly being a pain because eclipse kept offering me the wrong classpath for jena classes in the services module because of this.